### PR TITLE
feat: updated figma links in issue template

### LINKED ISSUE #10: Issue Template Updates for Figma link
### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

### Desired Situation
Each template that has a reference to Figma in its resources section should an invite link added.

### Implementation 
```
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Acceptance Tests**
All references to Figma include the "open invite" link.

### PR DESCRIPTION
**Pull request by abhijeetgauravm**
**Description**
Updated figma links in issue template.
This PR fixes #10 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ assignees: ''
 - Kubernetes Version: 
 - Host OS:
 - Browser: 
-
+- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
```

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge @abhijeetgauravm , thank you for your contribution :tada:

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Comment by leecalcote** [comment]
Thanks!

**Pull request merged**